mkosi.extra.common/usr/lib/systemd/system.conf.d/10-device-timeout.conf

```diff
@@ -3,4 +3,4 @@
 [Manager]
 # Lower the default device timeout so we get a shell earlier if the root device does
 # not appear for some reason.
-DefaultDeviceTimeoutSec=90
+DefaultDeviceTimeoutSec=120
```

Comment:
Hmm, this is a kind of optimization, if IIUC the comment. If this makes some tests unstable, then why not to simply drop this config?